app/services/current-user.js

```diff
@@ -59,8 +62,22 @@ export default Ember.Service.extend({
      */
     user: Ember.computed('currentUserId', function() {
         const ObjectPromiseProxy = Ember.ObjectProxy.extend(Ember.PromiseProxyMixin);
+        this.get('_setWaflle').perform();
```

Comment:
This sort of thing shouldn't go in a computed property, since it might not run if nothing is currently depending on `user`.

Instead, maybe subscribe to the session's [`authenticationSucceeded` and `invalidationSucceeded` events](http://ember-simple-auth.com/api/classes/SessionService.html#event_authenticationSucceeded) in the constructor?
```ts
this.get('session').on('authenticationSucceeded', this, function() {
    this.get('_setWaffle').perform();
});
```